packages/cfd/src/Containers/props.types.ts

```diff
@@ -162,14 +171,47 @@ export type TCFDPasswordManagerModal = {
 };
 
 export type TJurisdictionCardProps = {
+    account_status: GetAccountStatus;
+    account_type: string;
+    disabled: boolean;
     jurisdiction_selected_shortcode: string;
-    context: RootStore;
-    synthetic_available_accounts: TTradingPlatformAvailableAccount[];
-    financial_available_accounts: TTradingPlatformAvailableAccount[];
     setJurisdictionSelectedShortcode: (card_type: string) => void;
-    account_type: string;
-    type_of_card: string;
+    type_of_card: TJurisdictionCardType;
+};
+
+export type TJurisdictionCardBackProps = {
+    card_classname: string;
+    is_card_selected: boolean;
+    toggleCardFlip: (event: SyntheticEvent) => void;
```

Comment:
```suggestion
    toggleCardFlip: React.MouseEventHandler<HTMLSpanElement>;
```
please check this type for all toggleCardFlip's you find in your PR🙂

Reply:
@maryia-deriv since event is an Event and not an event handler, i am setting the type to React.MouseEvent.
event.stopPropogation needs to be invoked, which exist in MouseEvent.

Reply:
@shaheer-deriv 
`React.MouseEventHandler<HTMLSpanElement>` is just a shorter version of `(event: React.MouseEvent<HTMLSpanElement>) => void` 🙂 It's the same thing)